packages/cadl-ranch-specs/http/special-headers/conditional-request/main.tsp

```diff
@@ -43,3 +43,47 @@ op postIfNoneMatch(
   @doc("The request should only proceed if no entity matches this string.")
   ifNoneMatch?: string,
 ): NoContentResponse;
+
+@scenario
+@doc("""
+  Check when only If-Modified-Since in header is defined.
+  """)
+@scenarioDoc("""
+  Check when only If-Modified-Since in header is defined.
+  Expected header parameters:
+  - if-modified-since=Fri, 26 Aug 2022 14:38:00 GMT
+  """)
+@post
+@route("/if-modified-since")
+op postIfModifiedSince(
```

Comment:
Seems `if-modified-since` header can only appear in `GET` or `HEAD`:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Modified-Since

Reply:
Updated to use `HEAD`